packages/ui/src/components/pages/Accounts/__tests__/Accounts.spec.tsx

```diff
@@ -1,9 +1,9 @@
+import { waitFor } from '@testing-library/react';
```

Comment:
We could export `waitFor` from the `__tests__/testUtils` below, so we probably should move it there.